ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PauseResumeIntegrationTest.java

```diff
@@ -149,33 +149,39 @@ public void shouldPauseAndResumeMultipleQueries() {
     createQuery("2");
 
     Supplier<Integer> supplier = getSupplier(1);
+    Supplier<Integer> supplier2 = getSupplier(2);
     TEST_HARNESS.produceRows(PAGE_VIEW_TOPIC, PAGE_VIEWS_PROVIDER, KAFKA, JSON, System::currentTimeMillis);
 
     // 7 records are produced + a header & footer.
     assertThatEventually(supplier, equalTo(9));
+    assertThatEventually(supplier2, equalTo(9));
 
     // When:
     String queryId = ((Queries) RestIntegrationTestUtil.makeKsqlRequest(REST_APP, "SHOW "
         + "QUERIES;").get(0)).getQueries().get(0).getId().toString();
     RestIntegrationTestUtil.makeKsqlRequest(REST_APP, "PAUSE " + queryId + ";");
     assertThat(getPausedCount(), equalTo(1));
+    assertThat(getRunningCount(), equalTo(1));
 
     // Produce more records
     TEST_HARNESS.produceRows(PAGE_VIEW_TOPIC, PAGE_VIEWS_PROVIDER2, KAFKA, JSON,
         System::currentTimeMillis);
     
     assertThatEventually(supplier, equalTo(9));
+    assertThatEventually(supplier2, equalTo(14));
 
     // Then:
     RestIntegrationTestUtil.makeKsqlRequest(REST_APP, "RESUME " + queryId + ";");
 
     // 5 more records have been produced
     assertThatEventually(supplier, equalTo(14));
+    assertThatEventually(supplier2, equalTo(14));
 
     List <KsqlEntity> showQueries3 = RestIntegrationTestUtil.makeKsqlRequest(REST_APP, "SHOW "
         + "QUERIES;");
     assertThat(((Queries) showQueries3.get(0)).getQueries().get(0).getStatusCount().getStatuses()
         .get(KsqlQueryStatus.RUNNING), equalTo(1));
+    assertThat(getRunningCount(), equalTo(2));
```

Comment:
This looks like it was meant to replace the line above it?

Reply:
Good catch.